test/Microsoft.Extensions.ML.Tests/FileLoaderTests.cs

```diff
@@ -61,16 +59,14 @@ public async Task can_reload_model()
             var loaderUnderTest = ActivatorUtilities.CreateInstance<FileLoaderMock>(sp);
             loaderUnderTest.Start("testdata.txt", true);
 
-            var changed = false;
+            using AutoResetEvent changed = new AutoResetEvent(false);
             var changeTokenRegistration = ChangeToken.OnChange(
```

Comment:
>changeTokenRegistration [](start = 16, length = 23)

where is this being used?

Reply:
`changeTokenRegistration` is an `IDisposable`, so we should be disposing it. Fixing.